webflux6-micrometer/src/main/java/brave/example/Backend.java

```diff
@@ -4,13 +4,15 @@
 import java.util.Optional;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.context.annotation.Import;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestHeader;
 import org.springframework.web.bind.annotation.RestController;
 import reactor.core.publisher.Mono;
 
 @EnableAutoConfiguration
 @RestController
+@Import(ObservationConfiguration.class)
```

Comment:
Hm ... Could you please hint why you need it? The `spring-boot-actuator` is a dependency and `ObservationAutoConfiguration` should be picked by `@EnableAutoConfiguration`, no?

Reply:
should I change name to CustomObservationConfiguration?

Reply:
done!